textvalue-compose/src/main/kotlin/StringResources.kt

```diff
@@ -0,0 +1,29 @@
+package com.redmadrobot.textvalue
```

Comment:
I think it'll be better to change package here
```suggestion
package com.redmadrobot.textvalue.compose
```